src/Core/ObjectRegistry.php

```diff
@@ -208,26 +208,27 @@ public function has(string $name): bool
      * Get loaded object instance.
      *
      * @param string $name Name of object.
-     * @return object|null Object instance if loaded else null.
+     * @return object Object instance.
+     * @throws \RuntimeException If not loaded or found.
      */
-    public function get(string $name): ?object
+    public function get(string $name): object
     {
-        if (isset($this->_loaded[$name])) {
-            return $this->_loaded[$name];
+        if (!isset($this->_loaded[$name])) {
```

Comment:
Instead of negating the return of `isset()` could return the value here and throw exception after the `if` block.

Reply:
I can catch the value but the flow in code should always prioritize the happy case towards the end.